resources/public/api/conf/1.0/schemas/create_periodic_summary_request.json

```diff
@@ -0,0 +1,214 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Create a Self-Employment Period Summary",
+  "description": "Create a Self-Employment Period Summary Request",
+  "type": "object",
+  "properties": {
+    "periodDates": {
+      "type": "object",
+      "description": "Object containing the details about self-employment period dates.",
+      "properties": {
+        "periodStartDate": {
```

Comment:
periodStartDate descriptions in the spec state `Must conform to the format: YYYY-MM-DD` but the RAML appears as `Date in the format YYYY-MM-DD`